install/helm/agones/templates/ping.yaml

```diff
@@ -89,6 +89,22 @@ spec:
 {{- if .Values.agones.image.controller.pullSecret }}
       imagePullSecrets:
         - name: {{.Values.agones.image.controller.pullSecret}}
+{{- end }}
+{{- if .Values.agones.ping.pdb.enabled }}
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: agones-ping-pdb
+spec:
+  minAvailable: {{ .Values.agones.ping.pdb.minAvailable }}
+  maxUnavailable: {{ .Values.agones.ping.pdb.maxUnavailable }}
```

Comment:
According to https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget the above two fields are mutually exclusive (e.g. you can only set one or the other). 

It isn't clear to me what happens if the spec looks like:

```
spec:
  minAvailable: 2
  maxUnavailable: 
```

which is what will be produced if the boolean to enable this is toggled to true. Is that spec accepted by k8s or rejected?

Does the template here need to be more complicated so that we only set a single value?

Reply:
I was curious, so I tested this. It works fine, and sends the JSON you might expect: `"spec":{"maxUnavailable":null,"minAvailable":0, [...]}`.

Reply:
Thanks @zmerlynn for checking that out. Do you think this is ok to submit as-is then?

Reply:
I think it should still be documented, but we could also merge and come back to that later.